lib/active_merchant/billing/gateways/adyen_checkout.rb

```diff
@@ -348,21 +369,24 @@ def cvv_result_from(response)
         CVC_MAPPING[response['additionalData']['cvcResult'][0]] if response.dig('additionalData', 'cvcResult')
       end
 
-      def refund?(action)
-        action == "refund"
+      def should_use_pal_endpoint?(action)
+        action == "refund" || action == "disable"
       end
 
       def endpoint(action)
-        refund?(action) ? "Payment/#{PAL_API_VERSION}/#{action}" : "#{PAYMENTS_API_VERSION}/#{action}"
+        return "Payment/#{PAL_API_VERSION}/#{action}" if action == "refund"
```

Comment:
[non-blocking] wonder if rubocop complains if have all guards on top without separating them with ENTERS. It'd be more cohesive this way